### DISCUSSION TITLE
Using the GO Embed package for taxonomy/fybrik_application.json

### DISCUSSION THREAD

**Discussion by cdoron** [fybrik/fybrik#896] [Help]
In the Fybrik Manager code there is reference to "/tmp/taxonomy/fybrik_application.json".
Rather than reading the contents of this json from a file, we can load the contents to a variable. This could be done using the GO embed package: https://pkg.go.dev/embed

@ronenkat : what do you think?

**Comment by eletonia** [comment]
We are also referring to this file in the FybrikApplication webhook [here](https://github.com/fybrik/fybrik/blob/993fad413dc28a0d8537ff9fc425ea0c9d9dc0d6/manager/apis/app/v1alpha1/fybrikapplication_webhook.go#L32 )
